temp/ImageTemplateLinux.json

```diff
@@ -0,0 +1,226 @@
+{
+    "type": "Microsoft.VirtualMachineImages",
+    "apiVersion": "2022-02-14",
+    "location": "eastus",
+    "dependsOn": [],
+    "tags": {
+        "userIdentity": "enabled"
+    },
+    "identity": {
+        "type": "UserAssigned",
+        "userAssignedIdentities": {
+            "/subscriptions/a2b58cb8-c2cf-4afe-af64-63ed4dab8628/resourcegroups/AKSeVHDs/providers/Microsoft.ManagedIdentity/userAssignedIdentities/akse-e2e-uai": {}
```

Comment:
please remove hard-coded guids and resource names

Reply:
Removed